app/models/validator_history.rb

```diff
@@ -28,10 +28,11 @@
 #
 # Indexes
 #
-#  acceptable_stake_by_account_index                        (account,created_at,active_stake)
-#  delinquent_by_account_index                              (account,delinquent,created_at)
-#  index_validator_histories_on_network_and_account_and_id  (network,account,id)
-#  index_validator_histories_on_network_and_batch_uuid      (network,batch_uuid)
+#  acceptable_stake_by_account_index                                (account,created_at,active_stake)
+#  delinquent_by_account_index                                      (account,delinquent,created_at)
+#  index_validator_histories_on_network_and_account_and_id          (network,account,id)
+#  index_validator_histories_on_network_and_batch_uuid              (network,batch_uuid)
```

Comment:
If this index has been destroyed removed this entry, if it hasn't - please remove since index below is enough.